bigbluebutton-html5/imports/ui/components/whiteboard/styles.scss

```diff
@@ -43,7 +43,7 @@
 .whiteboardWrapper {
   order: 1;
   width: 100%;
-  height: 100%;
+  height: 90%; //I changed this
```

Comment:
You have cut 10% of the Whiteboard's height for everybody, even for viewers who will never see presenter controls.
That's not right, you should figure a way of how to shrink it only if a user sees presenter controls.